src/components/PrivateComponent/Private.component.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { useGlobal } from '../../providers/GlobalContext/GlobalContext';
+
+export default function Private({ children }) {
+  const globalContext = useGlobal();
+
+  return <>{globalContext.user ? children : ''}</>;
+}
```

Comment:
It sort of gets the job done, as it won't allow unauth users to view the children. But instead you should redirect the user to either Home or Login page using the `Redirect` from the router. Let me know if you have questions about this.